src/Spectre/Models/Msi/DivikResult/DivikConfig.cs

```diff
@@ -0,0 +1,108 @@
+﻿/*
+ * DivikConfug.cs
+ * Model class with divik config.
+ *
+   Copyright 2017 Grzegorz Mrukwa, Michał Gallus, Daniel Babiak
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+       http://www.apache.org/licenses/LICENSE-2.0
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.Serialization;
+using System.Web;
+
+namespace Spectre.Models.Msi.DivikResult
+{
+    /// <summary>
+    /// Provides divik config
+    /// </summary>
+    public class DivikConfig
```

Comment:
DivikOptions have some diff with config.json from divikResult, so I cannot parse it in easy way